stable/datadog/ci/cluster-agent-volumes-and-mounts.yaml

```diff
@@ -0,0 +1,12 @@
+clusterAgent:
+  enabled: true
+
+  volumes:
+    - name: secrets
+      secret:
+        secretName: test-secrets
+
+  volumeMounts:
+    - name: secrets
+      mountPath: /etc/secrets
```

Comment:
Could we also document the `"/etc/datadog-agent/certificates"` path to allow customers to mount certificates for the External Metrics Provider ?
https://github.com/DataDog/datadog-agent/blob/master/cmd/cluster-agent/custommetrics/server.go#L97

Reply:
So this is just a test here. Do you mean should add some more info about possible use cases to the `README.md`?